commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/commands/updateactions/ChangeTextLineItemQuantity.java

```diff
@@ -23,11 +23,11 @@ private ChangeTextLineItemQuantity(final String textLineItemId, final Long quant
         this.quantity = quantity;
     }
 
-    public static ChangeTextLineItemQuantity of(final String textLineItemId, final Long quantity) {
+    public static ChangeTextLineItemQuantity of(final String textLineItemId, final long quantity) {
```

Comment:
All primitive types were removed before v1.0.0. The reason was that sometimes a field became nullable (documentation error, evolution of the API...) and then the signature needed to change to the Object version, thus generating breaking changes for such a trivial thing.
Having a mix of primitive and objects depending on the resource felt a bit inconsistent too, but that's minor.

We could accept primitive types, keep a primitive type as field and return an Object. This way we gain in performance and yet keeping it breaking-changes-free. But we have to be careful to always keep it like this. But right now there is no gain in this code because we still keep the class field as Object.

So what do you think?